dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/RelativePeriods.java

```diff
@@ -922,6 +939,7 @@ public static Period setName( Period period, String periodName, boolean dynamicN
         add( list, RelativePeriodEnum.THIS_FINANCIAL_YEAR, thisFinancialYear );
         add( list, RelativePeriodEnum.LAST_FINANCIAL_YEAR, lastFinancialYear );
         add( list, RelativePeriodEnum.LAST_5_FINANCIAL_YEARS, last5FinancialYears );
+        add( list, RelativePeriodEnum.LAST_10_FINANCIAL_YEARS, last5FinancialYears );
```

Comment:
I believe there is a small mistake: "last5FinancialYears"

Reply:
fixed